src/omnicore/notifications.cpp

```diff
@@ -94,6 +94,7 @@ bool CheckAlertAuthorization(const std::string& sender)
     whitelisted.insert("16oDZYCspsczfgKXVj3xyvsxH21NpEj94F"); // Adam    <adam@omni.foundation>
     whitelisted.insert("1883ZMsRJfzKNozUBJBTCxQ7EaiNioNDWz"); // Zathras <zathras@omni.foundation>
     whitelisted.insert("1HHv91gRxqBzQ3gydMob3LU8hqXcWoLfvd"); // dexX7   <dexx@bitwatch.co>
+    whitelisted.insert("34kwkVRSvFVEoUwcQSgpQ4ZUasuZ54DJLD"); // multisig (signatories are Craig, Adam, Zathras, dexX7, JR)
```

Comment:
What is this? What's it redeem script? It's different from the activation multisig:

https://github.com/zathras-crypto/omnicore/blob/d630b68030e64a5866b6198af51f6530471bd3ef/src/omnicore/activation.cpp#L143

Reply:
It's the deactivation key (requires 3 sigs), the activation key requires 4.